monitor/getBalances.js

```diff
@@ -91,24 +113,38 @@ async function main(bridgeMode) {
     const blockRewardAddress = await homeBridge.methods.blockRewardContract().call()
     const blockRewardContract = new web3Home.eth.Contract(BLOCK_REWARD_ABI, blockRewardAddress)
     logger.debug('calling blockReward.methods.mintedTotally')
-    const mintedCoins = await blockRewardContract.methods.mintedTotally().call()
+    const mintedCoins = await blockRewardContract.methods.mintedTotallyByBridge(COMMON_HOME_BRIDGE_ADDRESS).call()
     logger.debug('calling homeBridge.methods.totalBurntCoins')
     const burntCoins = await homeBridge.methods.totalBurntCoins().call()
 
     const mintedCoinsBN = new BN(mintedCoins)
     const burntCoinsBN = new BN(burntCoins)
     const totalSupplyBN = mintedCoinsBN.minus(burntCoinsBN)
     const foreignErc20BalanceBN = new BN(foreignErc20Balance)
+    const halfDuplexErc20BalanceBN = new BN(foreignHalfDuplexErc20Balance)
```

Comment:
will it behave properly if the bridge does not support half duplex token?

Reply:
Yes, it will behave properly since the variable is initialized in `zero` if half duplex token is not supported yet so it won't have an impact in the diff. In the updated implementation it is more clear

Reply:
thanks!